### ISSUE TITLE
Groups Front End/UI/UX work

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/product-backlog#373]
As a part of the groups work, we will be redoing the top menu and adding new features that allow for new group navigation. @dawariley and others, we may want to break up this card, but I'm adding everything we have here to start.  
- [ ] Move sort and share icons down from top menu to the line below (so the icons share the same line as annotations/notes tabs)
- [ ] Create domain group scope structure
- [ ] Authoritative publishers domain that shows only on the publisher’s page
- [ ] Hypothesis groups that show on all pages. 
- [ ] Create a drop down menu that shows group control
- [ ] View group, and remove group (to start)
- [ ] Revised account control

**Comment by seanh** [comment]
@ajpeddakotla @dawariley I wonder if we can close this? I think we need github issues as individual user stories, I'd be worried that a "bag of UX changes" issue like this one will be difficult to get done in an iterative / sprint-based fashion.

* Move sort and share icons down from top menu to the line below (so the icons share the same line as  annotations/notes tabs) - sounds like we should make a separate issue?

* Create domain group scope structure - I've already made a [separate issue for this](https://github.com/hypothesis/product-backlog/issues/418)

* Authoritative publishers domain that shows only on the publisher’s page - I believe other groups user stories I've created like https://github.com/hypothesis/product-backlog/issues/417 and https://github.com/hypothesis/product-backlog/issues/418 will achieve this

* Hypothesis groups that show on all pages - This is how Hypothesis groups already work

* Create a drop down menu that shows group control - ?

* View group, and remove group (to start) - ?

* Revised account control - this doesn't seem to be strictly part of groups to me, but if it is I think we can make a separate issue
